dist/samples/store-locator/index.js

```diff
@@ -10,10 +12,21 @@ const stores = [];
 // Initialize and add the map
 function initMap() {
   distanceMatrixService = new google.maps.DistanceMatrixService();
+  originalMapTypeId = google.maps.MapTypeId.ROADMAP;
+  detailMapTypeId = google.maps.MapTypeId.HYBRID;
   map = new google.maps.Map(document.getElementById("map"), {
     center: { lat: 39.79, lng: -104.98 },
+    mapTypeId: originalMapTypeId,
     zoom: 10,
   });
+  map.addListener("zoom_changed", () => {
```

Comment:
would it make sense to use addListenerOnce inside of seeDetail for this?

Reply:
yes, that works better!